covasim/webapp/cova_app.css

```diff
@@ -20,6 +21,21 @@ html, body {
     grid-area: header;
 }
 
+.navbar {
+    background-color: #111313 !important;
+}
+
+.nav-text {
+    color: rgba(255,255,255,0.8) !important;
+}
+.nav-text:hover {
+    color: rgba(255,255,255,1.0) !important;
+}
```

Comment:
Overriding the nav-text class and adding the spans is unnecessary. Instead, just override the nav-link class.

Reply:
Added to issue